app/src/main/java/teamagam/ConnectionTest/Validator.java

```diff
@@ -0,0 +1,5 @@
+package teamagam.ConnectionTest;
+
+public interface Validator {
```

Comment:
As with Displayer, I don't find a good reason why not to put this interface into the ValidatorAsyncTask...